src/main/java/nextstep/subway/section/domain/Section.java

```diff
@@ -0,0 +1,89 @@
+package nextstep.subway.section.domain;
+
+import java.util.Objects;
+
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+import nextstep.subway.common.BaseEntity;
+import nextstep.subway.station.domain.Station;
+
+@Entity
+public class Section extends BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(nullable = false)
+    private Station up;
```

Comment:
```suggestion
    private Station upStation;
```
너무 축약하지 말고 명확하게 사용해보는것도 좋을 것 같아요~

참고해주시고, 사실 지금은 유지해주셔도 괜찮을 것 같습니다 !